src/ci.yml

```diff
@@ -62,19 +62,29 @@ jobs:
       {
         throw "UseCoreToolsBuildFromIntegrationTests can only be set to True or False. Current value is set to $env:USECORETOOLSBUILDFROMINTEGRATIONTESTS"
       }
-      ./setup-e2e-tests.ps1 -FunctionsRuntimeVersion $env:FUNCTIONSRUNTIMEVERSION `
+      ./setup-e2e-tests.ps1 -DotnetVersion $env:DOTNETVERSION `
                             -UseCoreToolsBuildFromIntegrationTests:$useIntegrationTestingCoreTools `
                             -SkipBuildOnPack
     displayName: "Setup E2E tests"
 
   - task: DotNetCoreCLI@2
+    condition: eq(variables['DotnetVersion'], 'net7')
     displayName: 'Run tests'
     inputs:
       command: 'test'
       arguments: '--no-build -c Release'
       projects: |
         test/**/*Tests.csproj
 
+  - task: DotNetCoreCLI@2
+    condition: eq(variables['DotnetVersion'], 'netfx')
+    displayName: 'Run tests for NetFramework'
+    inputs:
+      command: 'test'
+      arguments: '--no-build -c Release  --filter "FullyQualifiedName~TimerTrigger"' #only run http tests to avoid emulators on Linux
```

Comment:
Is the filter correct here? Did you mean "HttpTrigger" ?

Reply:
Did a bit of modifications to the PR so that such checks are not needed. Removed this condition now.